pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/BaseTransformFunction.java

```diff
@@ -92,7 +97,13 @@ public abstract class BaseTransformFunction implements TransformFunction {
   protected String[][] _stringValuesMV;
   protected byte[][][] _bytesValuesMV;
 
-  @Nullable
+  protected List<TransformFunction> _arguments;
+
+  // NOTE: this init has to be called for default getNullBitmap() implementation to be effective.
+  public void init(List<TransformFunction> arguments) {
```

Comment:
Suggest calling it `initArguments()` or `setArguments()` to avoid mixing it with the `TransformFunction.init()`, or override the `init()` with the same parameters

Reply:
done